.github/scripts/update_layer_arn.sh

```diff
@@ -44,7 +45,7 @@ for file in $files; do
     # Now replace the all "prefix"s in the file with the full new Layer ARN (line)
     # prefix:\d+ ==> line
     # sed doesn't support \d+ in a portable way, so we cheat with (:digit: :digit: *)
-    sed -i -e "s/$prefix:[[:digit:]][[:digit:]]*/$line/g" docs/index.md
+    sed -i -e "s/$prefix:[[:digit:]][[:digit:]]*/$line/g" $DOCS_FILE
```

Comment:
You don't have to do this now, but in Python I've simplified this by passing the new layer number to the workflow. Especially because you have the layer numbers in just one file.

All the complexity here is useful if we assume that the layer number can change between regions, which is not true in our case.

Check out this:
[aws-powertools/powertools-lambda-python@develop/layer_v3/scripts/update_layer_arn_v3.sh#L23-L33](https://github.com/aws-powertools/powertools-lambda-python/blob/develop/layer_v3/scripts/update_layer_arn_v3.sh#L23-L33)

Reply:
No, it's a good point - I thought the same when editing: this is needlessly over engineered.

I'll merge this today ahead of tomorrow's release but I'll also open an issue to replace this with a simpler version that takes the input as you suggested.

Reply:
Created #3890